README.md

```diff
@@ -146,8 +146,20 @@ module.exports = {
         test: /\.scss$/,
         use: ExtractTextPlugin.extract({
           fallback: 'style-loader',
-          //resolve-url-loader may be chained before sass-loader if necessary
-          use: ['css-loader', 'sass-loader']
+          use: [{
+              loader: 'css-loader',
+              options: {
+                  url: false, // If you are having trouble with urls not resolving add this setting.
```

Comment:
Please move the comment above `url: false` && add maybe attach a link to the `css-loader` README `// See https://github.com/webpack-contrib/css-loader#url`